KomaMRICore/src/KomaMRICore.jl

```diff
@@ -3,6 +3,7 @@ module KomaMRICore
 #IMPORT PACKAGES
 import Base.*, Base.+, Base.-, Base./, Base.vcat, Base.size, Base.abs, Base.getproperty
 #General
+using InteractiveUtils      # for using subtypes()
```

Comment:
You do not need to add this

Reply:
Addressed in [091441d](https://github.com/cncastillo/KomaMRI.jl/commit/091441dada08fa571bbf841d8018150cfc65f66e)